src/neoxp/Validators/TokenBase.cs

```diff
@@ -73,7 +73,8 @@ public bool IsValidSymbol()
         for (int i = 0; i < symbol.Length; i++)
         {
             var letter = symbol[i];
-            if ((letter >= 'A' && letter <= 'Z') == false)
+            if (char.IsWhiteSpace(letter) || char.IsControl(letter) ||
+                char.IsAscii(letter) == false)
                 return false;
         }
 
```

Comment:
```C#
return !symbol.Any(letter => char.IsWhiteSpace(letter) || 
                                 char.IsControl(letter) || 
                                 !char.IsAscii(letter));
```

Reply:
Ty, forgot about that. Cause i was doing `(letter >= 'A' && letter <= 'Z')` 😂